interface/main/tabs/menu/menus/standard.json

```diff
@@ -1238,14 +1238,32 @@
             "label":"Weno Management",
             "menu_id":"adm0",
             "target":"adm",
-            "url":"/interface/weno/admin.php",
+            "url":"/interface/weno/facilities.php",
             "children":[],
             "requirement":0,
             "acl_req": [
               "admin",
               "super"
-            ]
-          }
+            ],
+              "global_req": [
+                  "weno_rx_enable"
+              ]
+          },
+            {
+                "label":"Prescription Management",
+                "menu_id":"adm0",
+                "target":"adm",
+                "url":"/interface/weno/rxlogmanager.php",
+                "children":[],
+                "requirement":0,
+                "acl_req": [
+                    "super",
```

Comment:
change super to patients

Reply:
actually, if this is for admin stuff,then change it to admin and then place super below instead of rx

Reply:
If providers need to be able to see this, then agree with `patients` (note the s at the end), `rx` and then will need to allow this aco to be able to see the admin menu (lines 1268 and line 1275 )